examples/airline/airline.py

```diff
@@ -0,0 +1,19 @@
+from configs.agents import *
+from openai_runner.repl import run_demo_loop
+
+context_variables = {
+    "customer_context": """Here is what you know about the customer's details:
```

Comment:
Consider removing hardcoded customer details and implement a method to dynamically fetch or anonymize this data to enhance privacy and adaptability. [important]